h5py/_hl/files.py

```diff
@@ -327,8 +332,8 @@ def __init__(self, name, mode=None, driver=None,
             Name of the driver to use.  Legal values are None (default,
             recommended), 'core', 'sec2', 'stdio', 'mpio'.
         libver
-            Library version bounds.  Currently only the strings 'earliest'
-            and 'latest' are defined.
+            Library version bounds.  Supported values: 'earliest', 'v18' (HDF5
+            1.10.2 or later), 'v110' (HDF5 1.10.2 or later)  and 'latest'.
```

Comment:
It's a bit confusing that the version 1.8 marker is described as "1.10.2 or later". I know what you mean, but I bet people will try to 'fix' that.

Why don't we give the list, and then put a note afterwards like:

> The v18 and v110 options can only be specified on HDF5 1.10.2 or later.

Reply:
Agree.

Reply:
This PR has been quiet for some time until now. I don't see any issue beyond these two minor documentation fixes that you pointed out.

Reply:
Fixed in ff04733.